dev/release/verify-release-candidate.sh

```diff
@@ -1202,6 +1233,7 @@ test_wheels() {
 
 test_jars() {
   show_header "Testing Java JNI jars"
+  maybe_setup_maven
```

Comment:
Could you add an empty line between `show_header` and `maybe_setup_*` like others?

```suggestion

  maybe_setup_maven
```